content/en/architecture/component-overview.md

```diff
@@ -405,49 +405,6 @@ plgd cloud uses [NATS](https://nats.io) messaging system as it's event bus.
 
 ## Event Store
 
-plgd cloud persist events in an event store, which is a database of events. The store has an API for adding and retrieving device's events. Events needs to be versioned and written in a correct order. To achieve the consistency, optimistic concurrency control method is applied during each write.
-After the event is successfully written into the event store, it's distributed to the event bus which notifies all subscribers about the change asynchronously.
+The plgd Cloud stores each resource state mutation as a separate record called event. This enables us to reconstructs past states, use events as a foundation to understand user's behaviour or even explore an tamper-proof audit log. The plgd Cloud adopts [Event Sourcing](https://docs.microsoft.com/en-us/azure/architecture/patterns/event-sourcing) and [CQRS](https://docs.microsoft.com/en-us/azure/architecture/patterns/cqrs) patterns to support flexibility, scalability and trackability of the system.
```

Comment:
reconstructs -> reconstruct
an tamper-proof -> a tamper-proof

(I use https://languagetool.org/ to check grammar - articles, commas, etc.)

Reply:
Thanks 👍🏽